### PR TITLE
fix remove_objects() example to convert map to list of map

### PR DESCRIPTION
**Pull request by dsgibbons**
`Minio.remove_objects` expects a list of `DeleteObject`s, but the documentation uses a `map`. If a `map` is passed to `Minio.remove_objects`, it will silently fail, and the objects will not be deleted. This PR corrects the documentation to pass a `list` instead of a `map` to `Minio.remove_objects`.

### CODE REVIEW

File: minio/api.py [hunk 1]
```diff
@@ -2002,10 +2002,14 @@ def remove_objects(self, bucket_name, delete_object_list,
                 print("error occurred when deleting object", error)
 
             # Remove a prefix recursively.
-            delete_object_list = map(
-                lambda x: DeleteObject(x.object_name),
-                client.list_objects("my-bucket", "my/prefix/", recursive=True),
-            )
+            delete_object_list = [
+                DeleteObject(x).object_name
+                for x in client.list_objects(
```
Comment (balamurugana) [inline minio/api.py:2007]: What problem did you get when using `map()`?

**Review by balamurugana — CHANGES_REQUESTED** [review]

File: minio/api.py (hunk 1 above)
Comment (dsgibbons) [reply minio/api.py:2007]: `list(map(...))` works fine, but just using `map` doesn't work with `Minio.remove_objects`. This method expects a list. Passing a `map` fails silently and the objects are not removed.

In the above commits, I changed from `list(map(...))` to a list comprehension to make the code a bit more concise.

Comment (balamurugana) [reply minio/api.py:2007]: That is too verbose. Please make `list(map())`

Comment (dsgibbons) [reply minio/api.py]: Reverted to `list(map())`

**Review by balamurugana — APPROVED** [review]

**Pull request merged**